socorro/scripts/__init__.py

```diff
@@ -10,7 +10,17 @@ def _fill_text(self, text, width, indent):
         This makes it easier to do lists and paragraphs.
 
         """
-        parts = text.split('\n')
+        parts = text.split('\n\n')
         for i, part in enumerate(parts):
-            parts[i] = super(WrappedTextHelpFormatter, self)._fill_text(part, width, indent)
-        return '\n'.join(parts)
+            # Check to see if it's a bulleted list--if so, then fill each line
+            if part.startswith('* '):
+                subparts = part.split('\n')
+                for j, subpart in enumerate(subparts):
+                    subparts[j] = super(WrappedTextHelpFormatter, self)._fill_text(
+                        subpart, width, indent
+                    )
+                parts[i] = '\n'.join(subparts)
+            else:
+                parts[i] = super(WrappedTextHelpFormatter, self)._fill_text(part, width, indent)
+
+        return '\n\n'.join(parts)
```

Comment:
I pulled this out of a previous PR. It's helpful for printing out command line help that has lists.